init.sql

```diff
@@ -159,3 +159,202 @@ create table asset_creators
 create index asset_creator on asset_creators (asset_id, creator);
 create index asset_verified_creator on asset_creators (asset_id, verified);
+
+create type whitelist_mint_mode AS ENUM ('burn_every_time', 'never_burn');
+create type end_setting_type AS ENUM ('date', 'amount');
+
+create table candy_machine
+(
+    id                       bytea               PRIMARY KEY,
+    features                 int,
+    authority                bytea               not null,
+    mint_authority           bytea,
+    wallet                   bytea               not null,
+    token_mint               bytea,
+    items_redeemed           int                 not null,
+    candy_guard_pda          bytea,
+    version                  int                 not null,
+    collection_mint          bytea,                            
+    allow_thaw               bool,                                    
+    frozen_count             int,                                      
+    mint_start               int,
+    freeze_time              int,                                     
+    freeze_fee               int,                                      
+);
+
+create table candy_machine_data
+(
+    id                         bigserial        PRIMARY KEY,
+    uuid                       varchar(6),
+    price                      int,
+    symbol                     varchar(5)       not null,
+    seller_fee_basis_points    int              not null,
+    max_supply                 int              not null,
+    is_mutable                 bool             not null,
+    retain_authority           bool,
+    go_live_date               int,
+    items_available            int              not null,
+    candy_machine_id           bytea references candy_machine (id),
+);
+create unique index candy_machine_data_candy_machine_id on candy_machine_data (candy_machine_id);
+
+create table candy_machine_creators
+(
+    id                    bigserial                                PRIMARY KEY,
+    candy_machine_id      bytea references candy_machine (id)      not null,
+    creator               bytea                                    not null,
+    share                 int                                      not null default 0,
+    verified              bool                                     not null default false
+);
+create unique index candy_machine_creators_candy_machine_id on candy_machine_creators (candy_machine_id);
+create index candy_machine_creator on candy_machine_creators (candy_machine_id, creator);
+create index candy_machine_verified_creator on candy_machine_creators (candy_machine_id, verified);
+
+create table candy_guard
+(   
+    id                   bytea                                   PRIMARY KEY,
+    bump                 int                                     not null,
+    authority            bytea                                   not null,
+)
+
+create table candy_guard_group
```

Comment:
So I think the way you intend for this to work is that normally you will have a group with the "default" label to mimick the default GuardSet?  And then any other optional groups would just have different rows in this table with their specific custom labels, and you would join this ID with the different group table IDs to get the Guardset?

Reply:
so the way i had intended it to work was that i left label as optional. it will only be set on non default groups. that way my thinking on how to identify it

Reply:
comment added